changelogs/fragments/fix86_apply_tags.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - all roles - Fix tags to enable the execution of a specific tasks file as intended.
```

Comment:
This is not needed as the changelog is automatically generated from the PR title, please update the title with the relevant tag.
See here: https://github.com/prometheus-community/ansible/blob/main/CONTRIBUTING.md#releases